src/server/storage_engine/default_engine.rs

```diff
@@ -180,21 +200,27 @@ impl StorageEngine for DefaultEngine {
         }
 
         let local_file_name = generate_local_file_name(&self.root, &path);
-        let oflag = OFlag::O_RDONLY;
+        let oflag = OFlag::O_WRONLY;
```

Comment:
What will happen if write after reading? Will it return "read only fd"?

Reply:
It's a mistake.